### PR TITLE
fix imports and references

### PR DESCRIPTION
**Pull request by vuillaut**
My bad, we merged too soon.
Here is a quick fix.

### CODE REVIEW

File: lstchain/mc/plot_utils.py [hunk 1]
```diff
@@ -125,8 +130,8 @@ def sens_minimization_plot(eb, gb, tb, e, sens):
     Returns
     --------
     """
-    if (eb == 12):
-        figarr, axarr = plt.subplots(4,3, sharex=True, sharey=True, figsize=(13.2,18))
+    # if (eb == 12):
+    #     figarr, axarr = plt.subplots(4,3, sharex=True, sharey=True, figsize=(13.2,18))
 
     # To be changed!!!
```
Comment (rlopezcoto) [inline lstchain/mc/plot_utils.py]: could you move the comment up, before the commented lines? So I do not forget what is it that needs to be changed?

File: lstchain/mc/sensitivity.py [hunk 2]
```diff
@@ -62,7 +62,7 @@ def process_mc(simtelfile, dl2_file):
     events = pd.read_hdf(dl2_file)
 
     e_reco = 10**events.mc_energy * u.GeV
-    n_reco = e_reco.shape[0]
+    # n_reco = e_reco.shape[0]
     gammaness = events.gammaness
     theta2 = (events.src_x - events.src_x_rec)**2 + \
         (events.src_y - events.src_y_rec)**2 * u.deg**2
```
Comment (rlopezcoto) [inline lstchain/mc/sensitivity.py]: If you prefer to change the theta2 definition, please do so

File: lstchain/mc/sensitivity.py [hunk 3]
```diff
@@ -166,13 +166,13 @@ def sensitivity(emin_sens, emax_sens, eb, gb, tb, noff, obstime = 50 * 3600 * u.
...
 
-                ep_w_sum = np.sum(ep_reco_pw[(e_reco_p < E[i+1].to_value()) & (e_reco_p > E[i].to_value()) \
+                ep_w_sum = np.sum(e_reco_pw[(e_reco_p < E[i+1].to_value()) & (e_reco_p > E[i].to_value()) \
                                          & (gammaness_p > g[j]) & (theta2_p < t[k])])
             
                 final_gamma[i][j][k] = eg_w_sum * obstime
                 final_hadrons[i][j][k] = ep_w_sum * obstime
                     
-    sens = calculate_sensititity(final_gamma.to_value(), final_hadrons.to_value(), 1/noff)
+    sens = calculate_sensitivity(final_gamma.to_value(), final_hadrons.to_value(), 1/noff)
```
Comment (rlopezcoto) [inline lstchain/mc/sensitivity.py]: That one was very bad, thanks! ;-)

File: lstchain/mc/plot_utils.py (hunk 1 above)
Comment (vuillaut) [reply lstchain/mc/plot_utils.py]: done

File: lstchain/mc/sensitivity.py (hunk 2 above)
Comment (vuillaut) [reply lstchain/mc/sensitivity.py]: Will do in a future PR.
I'd like to introduce the use of other libraries for common stuff like this.

Comment (vuillaut) [reply lstchain/mc/sensitivity.py]: I just wanted to make a quick fix here.

Comment (rlopezcoto) [reply lstchain/mc/sensitivity.py]: which other libraries? 
well we can discuss about this later, but I would try not to make it too complex if the computing time overhead is too large.

Comment (vuillaut) [reply lstchain/mc/sensitivity.py]: We are here trying to re-implement many things that are already implemented in `astropy`, `gammapy` or `ctaplot` which means maintenance that we should avoid (and potentially also means wrong implementations).
But we can discuss that later.

**Comment by vuillaut** [comment]
@rlopezcoto can we merge this small patch to have at least working functions?
As discussed in pr #105, improvements on all that shall be coming later.

**Pull request merged**